src/modules/workspaces/repositories/team.repository.spec.ts

```diff
@@ -129,25 +134,63 @@ describe('SequelizeWorkspaceTeamRepository', () => {
     });
   });
 
-  describe('getTeamsInWorkspace', () => {
-    const workspace = newWorkspace();
-    it('When a workspace id is not found then we receive an empty array', async () => {
-      jest.spyOn(workspaceTeamModel, 'findAll').mockResolvedValueOnce([]);
-      const response = await repository.getTeamsInWorkspace(workspace.id);
-      expect(response).toEqual([]);
+  describe('getTeamAndMemberByWorkspaceAndMemberId', () => {
+    it('When members are not found, then return empty array', async () => {
+      jest.spyOn(workspaceTeamUserModel, 'findAll').mockResolvedValueOnce([]);
+
+      const result = await repository.getTeamAndMemberByWorkspaceAndMemberId(
+        'workspaceiD',
+        'memberId',
+      );
+
+      expect(result).toEqual([]);
     });
 
-    it('When a workspace id is found then we receive a WorkspaceTeam[]', async () => {
-      const team1 = newWorkspaceTeam({ workspaceId: workspace.id });
-      const team2 = newWorkspaceTeam({ workspaceId: workspace.id });
+    it('When members are found, then return member data and team', async () => {
+      const teamRaw = newWorkspaceTeam();
+      const workspaceTeamUserRaw = newWorkspaceTeamUser({ teamId: teamRaw.id });
+
+      jest.spyOn(workspaceTeamUserModel, 'findAll').mockResolvedValueOnce([
+        {
+          ...workspaceTeamUserRaw.toJSON(),
+          toJSON: jest.fn().mockReturnValue(workspaceTeamUserRaw),
+          team: {
+            ...teamRaw.toJSON(),
+            toJSON: jest.fn().mockReturnValue(teamRaw),
+          },
+        },
+      ] as any);
 
-      jest
-        .spyOn(workspaceTeamModel, 'findAll')
-        .mockResolvedValueOnce([team1, team2] as any);
+      const result = await repository.getTeamAndMemberByWorkspaceAndMemberId(
+        'workspaceiD',
+        'memberId',
+      );
 
-      const response = await repository.getTeamsInWorkspace(workspace.id);
+      expect(result).toEqual([
+        { team: teamRaw, teamUser: workspaceTeamUserRaw },
+      ]);
+    });
+
+    describe('getTeamsInWorkspace', () => {
+      const workspace = newWorkspace();
+      it('When a workspace id is not found then we receive an empty array', async () => {
+        jest.spyOn(workspaceTeamModel, 'findAll').mockResolvedValueOnce([]);
+        const response = await repository.getTeamsInWorkspace(workspace.id);
+        expect(response).toEqual([]);
+      });
+
+      it('When a workspace id is found then we receive a WorkspaceTeam[]', async () => {
```

Comment:
Why the legend is using []? In any case, we can say, 'We receive the workspace's teams list'